src/lib.rs

```diff
@@ -17,8 +18,12 @@ pub type AccountId = AccountId32;
 pub type CodeHash = H256;
 /// An alias for a block hash type.
 pub type BlockHash = H256;
+/// An alias for a block number type.
+pub type BlockNumber = u32;
 /// An alias for a transaction hash type.
 pub type TxHash = H256;
+/// An alias for the amount of tokens.
+pub type Balance = u128;
```

Comment:
How generic we want this lib to be? Should these types be configurable maybe?

Reply:
good spot; I divided types into:
 - pallet specific: these will be defined on the pallet-related traits level (as in this PR)
 - chain general (as these in `lib.rs`): until we migrate everything in our codebase, I want to leave them as they are now (hardcoded); then I want to try get some feedback from other projects and check if there is a need of abstracting these types as well

Reply:
OK, makes sense